src/options/modules/ColorSchemeModeHelper.js

```diff
@@ -9,6 +14,7 @@ export function changeScreenshotTheme(darkQuery){
     } else {
         document.getElementById("searchBarDemo").src = "./img/emojiSearchDog_light.png";
     }
+    return ColorSchemeModeHelper.changeScreenshotTheme(darkQuery);
```

Comment:
Did you actually test this change? (pls always do so)

Because AFAIK this should not work.

Please read https://hacks.mozilla.org/2015/08/es6-in-depth-modules/ or similar in order to understand how these modules work.

Reply:
I tested it on Firefox dev edition by changing my OS theme and yes it worked fine. Are we talking about some other tests?

Reply:
Well, no other tests, just usual dev testing.

Because it may work, but not without side-effects. Here is an exception thrown if you look into the debugger:
![exception](https://user-images.githubusercontent.com/11966684/68528247-5afa9680-02f0-11ea-9c76-05954d6bfe2f.png)